packages/decoupled-logger/tsconfig.base.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
@friendken Please see if we can share those config-files and build-scripts by using symlinks so we can avoid duplicating them for every package.